roles/ansible_repositories/tasks/redhat.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: 'Install centos-release-ansible29 on CentOS8'  # noqa package-latest
+  dnf:
+    name: centos-release-ansible-29
+    state: latest
+    update_cache: true
+  tags:
+    - packages
+  when:
+    - ansible_distribution == 'CentOS'
+    - ansible_distribution_major_version == '8'
+
+- name: 'Install centos-release-ansible29 on Rocky8'
+  dnf:
+    name:
+      - http://mirror.centos.org/centos/8/extras/x86_64/os/Packages/centos-release-configmanagement-1-1.el8.noarch.rpm
+      - http://mirror.centos.org/centos/8/extras/x86_64/os/Packages/centos-release-ansible-29-1-2.el8.noarch.rpm
+    disable_gpg_check: true
+    state: present
+  tags:
+    - packages
+  when:
+    - ansible_distribution == 'Rocky'
+    - ansible_distribution_major_version == '8'
```

Comment:
according to https://www.reddit.com/r/RockyLinux/comments/pny121/rocky_linux_addons_rockyrelease_centos_sig_content/, you should just be able to get the centos release bits from rocky extras?

Reply:
and the same for Alma: https://www.reddit.com/r/AlmaLinux/comments/pg4ev7/announcing_centos_sig_repository_availability_in/

Reply:
Oh nice! I didn't know and it wasn't in place when I first wrote this :)

I have updated the task to 'just' install centos-release-ansible-29 now :)